scripts/config.pl

```diff
@@ -46,7 +46,7 @@
 my $usage = <<EOU;
 $0 [-f <file> | --file <file>] [-o | --force]
                    [set <symbol> <value> | unset <symbol> | get <symbol> |
-                        full | realfull | baremetal]
+                        full | realfull | baremetal | all]
```

Comment:
Adding this option was necessary to automate the generation of the query_config.c file because the previous options did not uncomment the module defines in the config.h